charts/testkube-cloud-api/values.yaml

```diff
@@ -201,6 +201,12 @@ scim:
     # -- Secret ref for SCIM server authentication (secret must contain key SCIM_AUTH_TOKEN)
     secretRef: ""
 api:
+  dex:
```

Comment:
Please also duplicate this to the `values.yaml` file of the estkube-enterprise chart

Reply:
This currently isn't really meant to be used in Enterprise, this is currently only for Cloud